from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_login_window(object):
    def setupUi(self, login_window):
        login_window.setObjectName("login_window")
        login_window.resize(283, 193)
        login_window.setMinimumSize(QtCore.QSize(283, 193))
        login_window.setMaximumSize(QtCore.QSize(283, 193))
        self.layoutWidget = QtWidgets.QWidget(parent=login_window)
        self.layoutWidget.setGeometry(QtCore.QRect(10, 10, 265, 171))
        self.layoutWidget.setObjectName("layoutWidget")
        self.verticalLayout_4 = QtWidgets.QVBoxLayout(self.layoutWidget)
        self.verticalLayout_4.setContentsMargins(0, 0, 0, 0)
        self.verticalLayout_4.setObjectName("verticalLayout_4")
        self.verticalLayout_2 = QtWidgets.QVBoxLayout()
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.main_label = QtWidgets.QLabel(parent=self.layoutWidget)
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(12)
        font.setBold(True)
        font.setWeight(75)
        self.main_label.setFont(font)
        self.main_label.setScaledContents(False)
        self.main_label.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.main_label.setWordWrap(True)
        self.main_label.setObjectName("main_label")
        self.verticalLayout_2.addWidget(self.main_label)
        self.horizontalLayout = QtWidgets.QHBoxLayout()
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.verticalLayout = QtWidgets.QVBoxLayout()
        self.verticalLayout.setObjectName("verticalLayout")
        self.login_input = QtWidgets.QLineEdit(parent=self.layoutWidget)
        self.login_input.setText("")
        self.login_input.setObjectName("login_input")
        self.verticalLayout.addWidget(self.login_input)
        self.pass_input = QtWidgets.QLineEdit(parent=self.layoutWidget)
        self.pass_input.setText("")
        self.pass_input.setEchoMode(QtWidgets.QLineEdit.EchoMode.Password)
        self.pass_input.setObjectName("pass_input")
        self.verticalLayout.addWidget(self.pass_input)
        self.horizontalLayout.addLayout(self.verticalLayout)
        self.login_button = QtWidgets.QPushButton(parent=self.layoutWidget)
        font = QtGui.QFont()
        font.setBold(True)
        font.setWeight(75)
        self.login_button.setFont(font)
        self.login_button.setObjectName("login_button")
        self.horizontalLayout.addWidget(self.login_button)
        self.verticalLayout_2.addLayout(self.horizontalLayout)
        self.verticalLayout_4.addLayout(self.verticalLayout_2)
        self.verticalLayout_3 = QtWidgets.QVBoxLayout()
        self.verticalLayout_3.setObjectName("verticalLayout_3")
        self.contact_button = QtWidgets.QPushButton(parent=self.layoutWidget)
        font = QtGui.QFont()
        font.setItalic(True)
        self.contact_button.setFont(font)
        self.contact_button.setObjectName("contact_button")
        self.verticalLayout_3.addWidget(self.contact_button)
        self.verticalLayout_4.addLayout(self.verticalLayout_3)

        self.retranslateUi(login_window)
        QtCore.QMetaObject.connectSlotsByName(login_window)

    def retranslateUi(self, login_window):
        _translate = QtCore.QCoreApplication.translate
        login_window.setWindowTitle(_translate("login_window", "Авторизация в ЦБД ГАИ Челябинск"))
        self.main_label.setText(
            _translate("login_window", "Централизованная база данных Госавтоинспекции УМВД России по г. Челябинску"))
        self.login_input.setPlaceholderText(_translate("login_window", "Введите логин пользователя"))
        self.pass_input.setPlaceholderText(_translate("login_window", "Введите пароль"))
        self.login_button.setText(_translate("login_window", "Вход"))
        self.contact_button.setText(_translate("login_window", "Написать разработчику в Telegram"))
